src/main/webapp/app/exercises/programming/assess/repo-export/programming-assessment-repo-export-dialog.component.ts

```diff
@@ -75,16 +76,18 @@ export class ProgrammingAssessmentRepoExportDialogComponent implements OnInit {
         if (this.participationIdList) {
             // We anonymize the assessment process ("double-blind").
             this.repositoryExportOptions.addParticipantName = false;
-            this.repoExportService.exportReposByParticipations(exerciseId, this.participationIdList, this.repositoryExportOptions).subscribe(this.handleExportRepoResponse, () => {
-                this.exportInProgress = false;
-            });
+            this.repoExportService
+                .exportReposByParticipations(exerciseId, this.participationIdList, this.repositoryExportOptions, this.hideStudentName)
```

Comment:
What are the export options? Hiding a student name sounds like an export option.

Reply:
good catch! I just refactored it to use the export options